data/formats-data.js

```diff
@@ -4662,7 +4669,7 @@ exports.BattleFormatsData = {
 	},
 	krookodile: {
 		randomBattleMoves: ["earthquake", "stoneedge", "pursuit", "knockoff", "stealthrock", "superpower"],
-		randomDoubleBattleMoves: ["earthquake", "stoneedge", "protect", "knockoff", "superpower"],
+		randomDoubleBattleMoves: ["earthquake", "stoneedge", "protect", "knockoff", "superpower", "firefang"],
```

Comment:
Why does Krook need Fire coverage? Its other moves cover it well enough and it doesn't even have Strong Jaw